test/plugin/test_in_windows_eventlog2.rb

```diff
@@ -226,6 +226,36 @@ def test_parse_desc
     assert_equal(expected, h)
   end
 
+  def test_parse_desc_camelcase
+    d = create_driver(config_element("ROOT", "", {"tag" => "fluent.eventlog",
+                                                  "parse_description" => true,
+                                                  "description_key_delimiter" => "",
+                                                  "description_word_delimiter" => "",
+                                                  "downcase_description_keys" => false
+                                                  }, [
+                                       config_element("storage", "", {
+                                                        '@type' => 'local',
+                                                        'persistent' => false
+                                                      }),
+                                     ]))
+    desc =<<-DESC
+A user's local group membership was enumerated.\r\n\r\nSubject:\r\n\tSecurity ID:\t\tS-X-Y-XX-WWWWWW-VVVV\r\n\tAccount Name:\t\tAdministrator\r\n\tAccount Domain:\t\tDESKTOP-FLUENTTEST\r\n\tLogon ID:\t\t0x3185B1\r\n\r\nUser:\r\n\tSecurity ID:\t\tS-X-Y-XX-WWWWWW-VVVV\r\n\tAccount Name:\t\tAdministrator\r\n\tAccount Domain:\t\tDESKTOP-FLUENTTEST\r\n\r\nProcess Information:\r\n\tProcess ID:\t\t0x50b8\r\n\tProcess Name:\t\tC:\\msys64\\usr\\bin\\make.exe
+DESC
+    h = {"Description" => desc}
+    expected = {"DescriptionTitle"                 => "A user's local group membership was enumerated.",
+                "SubjectSecurityId"                => "S-X-Y-XX-WWWWWW-VVVV",
+                "SubjectAccountName"               => "Administrator",
+                "SubjectAccountDomain"             => "DESKTOP-FLUENTTEST",
+                "SubjectLogonId"                   => "0x3185B1",
+                "UserSecurityId"                   => "S-X-Y-XX-WWWWWW-VVVV",
+                "UserAccountName"                  => "Administrator",
+                "UserAccountDomain"                => "DESKTOP-FLUENTTEST",
+                "ProcessInformationProcessId"      => "0x50b8",
```

Comment:
Thanks! Basically looks good!
We need to fix the following to make the test succeed.

```suggestion
                "SubjectSecurityID"                => "S-X-Y-XX-WWWWWW-VVVV",
                "SubjectAccountName"               => "Administrator",
                "SubjectAccountDomain"             => "DESKTOP-FLUENTTEST",
                "SubjectLogonID"                   => "0x3185B1",
                "UserSecurityID"                   => "S-X-Y-XX-WWWWWW-VVVV",
                "UserAccountName"                  => "Administrator",
                "UserAccountDomain"                => "DESKTOP-FLUENTTEST",
                "ProcessInformationProcessID"      => "0x50b8",
```

Reply:
Nice catch!